pom.xml

```diff
@@ -581,6 +615,11 @@
         <artifactId>awaitility</artifactId>
         <version>${awaitility.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.codehaus.plexus</groupId>
+        <artifactId>plexus-classworlds</artifactId>
+        <version>2.8.0</version>
```

Comment:
No version property?

Reply:
yup I could (I will for consistency) 
But sometimes I wonder why are we doing this. properties are nice to avoid repeating but here most of them are just a single location :).